src/webpki.rs

```diff
@@ -108,17 +109,27 @@ pub struct EndEntityCert<'a> {
     inner: cert::Cert<'a>,
 }
 
-impl<'a> EndEntityCert<'a> {
+impl<'a> TryFrom<&'a [u8]> for EndEntityCert<'a> {
```

Comment:
NIT: In the other implementation you used `impl ... core::convert::TryFrom` and here you used `use ::cure::convert::TryFrom` and then the `impl` is unqualified. I think we should be consistent.